### PR TITLE
[Site-313] Adjust the appendix present on every article page bottom

### PR DESCRIPTION
**Pull request by nenispng**
**Link to issue 🔗** 
https://labcodes.atlassian.net/browse/SITE-313

**What's new? ✨** 
- text, link and tag adjustments on newsletter invite
- margin, color and width adjustments on CTA
- width adjustment on disqus widget

### CODE REVIEW

File: themes/medius/templates/includes/call-to-action.html [hunk 1]
```diff
@@ -1,9 +1,9 @@
 <aside class="call-to-action">
   <div class="box text">
     <h2>
-      Have an exciting idea? <br >
-      Let's build something great together! <br>
-      We're thrilled to hear about your project. <br>
+      Have an exciting idea? </br>
+      Let's build something great together! </br>
+      We're thrilled to hear about your project. </br>
```
Comment (fernandolins) [inline themes/medius/templates/includes/call-to-action.html:6]: Why did you add a slash to br html tag?
https://www.w3schools.com/tags/tag_br.asp

File: themes/medius/templates/includes/article-footer.html [hunk 2]
```diff
@@ -1,9 +1,15 @@
+</br>
```
Comment (fernandolins) [inline themes/medius/templates/includes/article-footer.html:1]: Why did you add a slash to br html tag?
https://www.w3schools.com/tags/tag_br.asp

File: themes/medius/templates/includes/article-footer-pt.html [hunk 3]
```diff
@@ -1,9 +1,16 @@
+</br>
```
Comment (fernandolins) [inline themes/medius/templates/includes/article-footer-pt.html:1]: Why did you add a slash to br html tag?
https://www.w3schools.com/tags/tag_br.asp

File: themes/medius/scss/shared.scss [hunk 4]
```diff
@@ -25,60 +17,74 @@
...
-      padding: 40px 20px;
-      background-color: $coral;
-      letter-spacing: -0.01em;
-      font-size: $fs-subtitle;
-
-      @media (min-width: 768px) {
-        padding: 20px 100px;
-      }
+      padding: $spacing-7;
+      background-color: $purple;
```
Comment (fernandolins) [inline themes/medius/scss/shared.scss:22]:
```suggestion
      background-color: $purple-60;
```

**Review by fernandolins — CHANGES_REQUESTED** [review]

File: themes/medius/templates/includes/call-to-action.html (hunk 1 above)
Comment (nenispng) [reply themes/medius/templates/includes/call-to-action.html]: I have mistaken html with xhtml. Already commited the fixes.

File: themes/medius/templates/includes/article-footer.html (hunk 2 above)
Comment (nenispng) [reply themes/medius/templates/includes/article-footer.html]: I have mistaken html with xhtml. Already commited the fixes.

File: themes/medius/templates/includes/article-footer-pt.html (hunk 3 above)
Comment (nenispng) [reply themes/medius/templates/includes/article-footer-pt.html]: I have mistaken html with xhtml. Already commited the fixes.

**Review by fernandolins — APPROVED** [review]
LGTM

**Pull request merged**